mkosi

```diff
@@ -1467,14 +1467,14 @@ def clean_package_manager_metadata(workspace: str) -> None:
     # FIXME: implement cleanup for other package managers
 
 def clean_tdnf_metadata(root: str) -> None:
-	    """Removes tdnf metadata iff /bin/tdnf is not present in the image"""
-	    tdnf_path = root + '/usr/bin/tdnf'
-	    keep_tdnf_data = os.access(tdnf_path, os.F_OK, follow_symlinks=False)
+        """Removes tdnf metadata iff /bin/tdnf is not present in the image"""
+        tdnf_path = root + '/usr/bin/tdnf'
+        keep_tdnf_data = os.access(tdnf_path, os.F_OK, follow_symlinks=False)
```

Comment:
That's not 4 spaces, is it?

Reply:
No, it  is not, the original was neither one tab, neither 4 spaces but tab and 4 spaces.
I changed tabs without realigning (on purpose), but is you prefer (and I prefer also that, by the way) I suppress all redundant idents, let me know, I'll manage it before it to be merged (I have not done it since I did not know the feeling about the indentation problem), I've done that so it was pretty clear only tabs where changed.